modules/robin/src/main/ts/ephox/robin/zone/LanguageZones.ts

```diff
@@ -122,9 +122,10 @@ const nu = <E>(defaultLang: string): LanguageZones<E> => {
 //    (regardless of 'classic'/iframe or 'inline'/div mode).
 // Note: there may be descendant elements with a different language
 const calculate = <E, D>(universe: Universe<E, D>, item: E): Optional<string> => {
-  return universe.up().closest(item, '[lang]', Fun.never).bind((el) => {
-    const lang = universe.attrs().get(el, 'lang');
-    return lang === undefined ? Optional.none<string>() : Optional.some(lang);
+  const getLanguage = universe.property().getLanguage;
+  return getLanguage(item).orThunk(() => {
+    const ancestors = universe.up().all(item, Fun.never);
+    return Arr.findMap(ancestors, getLanguage);
```

Comment:
```suggestion
  const props = universe.property();
  return props.getLanguage(item).orThunk(() => {
    const ancestors = universe.up().all(item, Fun.never);
    return Arr.findMap(ancestors, props.getLanguage);
```
This is rather strange to be storing a variable for a function and not helpful for debugging either. So just trying to change it to be more common with what might be done elsewhere, though I know it won't minify quite as well so not entirely sure about this...